Autopilot/Path Manager/PathManager.c

```diff
@@ -667,6 +674,19 @@ void copyGPSData(){
     pmData.checkbyteDMA2 = generatePMDataDMAChecksum2();
 }
 
+static void checkForFirstGPSLock(){
+	static char gpsLockFlag = 1;
+	
+	if(((getErrorCodes() & BOR_RESET) || (getErrorCodes() & POR_RESET)) && gpsLockFlag){
```

Comment:
I would just move your `gpsLockFlag` variable outside the function, which will take care of the static problem. Calling the function twice isnt a very big deal, so you can keep that. Otherwise you can just use a nested if:
```c
if (gpsLockFlag){
    char error_codes = getErrorCodes();
    if (error_codes & BOR_RESET || error_codes & POR_RESET){
     //your logic
    }
}
```

```